spec/models/team_spec.rb

```diff
@@ -7,4 +7,9 @@
   it { should validate_presence_of(:owner) }
   it { should have_many(:namespaces) }
 
+  it 'downcases the name on creation' do
+    team = FactoryGirl.create(:team, name: 'TeAm')
```

Comment:
Does it make more sense to perform that action before validation?

That way:

* you first normalize data, and then validate normalized
* in after create you need to follow that flow:
1) validate - is it valid? yes
2) go to `downcase!`  
3) save - no validation!

Hence - if you method is providing non valid data - it will be blindly saved 

I would go for introducing `validator` for the team name - it should be down-cased and format is there in registry 2.0 docs

that way even the need to create team will drop off - you can do that via build